docs/BR.md

```diff
@@ -1739,142 +1737,917 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+      * [Section 7.1.2.5 - TLS Subordinate CA Certificate Profile](#7125-tls-subordinate-ca-certificate-profile)
+  * [Section 7.1.2.6 - Subscriber (End-Entity) Certificate Profile](#7126-subscriber-server-certificate-profile)
+    * Domain Validated
+    * Individual Validated
+    * Organization Validated
+    * Extended Validation
+  * __**TBD: Infrastructure Certificates**__
+    * [Section 7.1.2.7 - OCSP Responder Certificate Profile](#7127-ocsp-responder-certificate-profile)
+    * __**TBD: Infrastructure Cert**__
+    * __**TBD: Precert Signing CA**__
+    * __**TBD: Interoperability Testing Certs (valid/revoked/expired)?**__
+
+#### 7.1.2.1 Root CA Certificate Profile
+
+| __Field__                      | __Description__ |
+| ---                            | ------          |
+| `tbsCertificate`               | |
+| \ \ \ \ `version`              | MUST be v3(2) |
+| \ \ \ \ `serialNumber`         | MUST be a non-sequential number greater than zero (0) containing at least 64 bits of output from a CSPRNG. |
+| \ \ \ \ `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
+| \ \ \ \ `issuer`               | Encoded value MUST be byte-for-byte identical to the encoded `subject` |
+| \ \ \ \ `validity`             | __**TBD**__ |
+| \ \ \ \ `subject`              | See [Section 7.1.2.8.1](#71281-ca-certificate-naming) |
+| \ \ \ \ `subjectPublicKeyInfo` | See [Section 7.1.3.1](#7131-subjectpublickeyinfo) |
+| \ \ \ \ `issuerUniqueID`       | MUST NOT be present |
+| \ \ \ \ `subjectUniqueID`      | MUST NOT be present |
+| \ \ \ \ `extensions`           | See [Section 7.1.2.1.1](#71211-root-ca-extensions) |
+| `signatureAlgorithm`           | Encoded value MUST be byte-for-byte identical to the `tbsCertificate.signature`. |
+| `signature`                    | |
+
+##### 7.1.2.1.1 Root CA Extensions
+
+| __Extension__                     | __Presence__ | __Critical__ | __Description__ |
+| ----                              | -            | -            | ----- |
+| `authorityKeyIdentifier`          | SHOULD       | N            | See [Section 7.1.2.1.2](#71212-authority-key-identifier) |
+| `basicConstraints`                | MUST         | Y            | See [Section 7.1.2.1.3](#71213-basic-constraints) |
+| `keyUsage`                        | MUST         | Y            | See [Section 7.1.2.8.8](#71288-key-usage) |
+| `subjectKeyIdentifier`            | MUST         | N            | See [Section 7.1.2.9.4](#71294-subject-key-identifier) |
+| `extKeyUsage`                     | MUST NOT     | N            | - |
+| `certificatePolicies`             | SHOULD NOT   | N            | See [Section 7.1.2.8 4](#71284-certificate-policies---affiliated-ca) |
+| Signed Certificate Timestamp List | MAY          | N            | See [Section 7.1.2.9.3](#71293-signed-certificate-timestamp-list) |
+| Any other extension               | SHOULD NOT   | -            | __**TBD**__ |
```

Comment:
Microsoft Certificate Services inserts a few extensions by default denoting several pieces of metadata, and quite a few CAs have CA certificates with these included. Either they should be explicitly listed here, or the SHOULD NOT be relaxed, or work with Microsoft to better allow for them to be suppressed in webPKI certificates.

Reply:
I don't think that changes the SHOULD NOT - such metadata SHOULD NOT be included. CAs are still permitted to, but it indeed signals the exact process you propose - that they work with Microsoft to better understand how to use ADCS certificate profiles to remove that metadata ;)

This only applies to new issuance, as with all profiles, so I don't see that being a fundamental problem until we (eventually, hopefully within 6 months to 1 year) sunset this to a MUST NOT.

I don't see how this is different than any other COTS PKI.